e2e-tests/init-deploy/compare/max_allowed_packet-2-80.sql

```diff
@@ -0,0 +1 @@
+827326464
```

Comment:
I think this file identical to `e2e-tests/init-deploy/compare/max_allowed_packet-2.sql`, so it is not needed.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/337/commits/91639c5fde00ad69ef8ad91208ebda0f0f55088d